app/views/TrustNotClaimedView.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    main_template: MainTemplate
+)
+
+@(utr: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("trustNotClaimed.title")
+    ) {
+
+    @components.back_link()
+
+    <span class="govuk-caption-xl">@messages("trustNotClaimed.subheading", utr)</span>
+    @components.heading("trustNotClaimed.heading")
+
+    <p>@messages("trustNotClaimed.p1")</p>
+
+    <ol class="govuk-list--number">
+        <li class="bold">@messages("trustNotClaimed.bullet.1.title")</li>
+        <p>@messages("trustNotClaimed.bullet.1.p1")</p>
+        <p>@messages("trustNotClaimed.bullet.1.p2") <a target="_blank" href="https://www.tax.service.gov.uk/government-gateway-registration-frontend/choose-your-account?continue=%2Ftrusts-forms%2Fform%2Fregistration-of-a-trust%2Fnew&origin=unknown">@messages("trustNotClaimed.bullet.1.p2.a")</a></p>
```

Comment:
This is the link I'm unsure about.
We're referring to a production URL in all environments, and we currently haven't sent a user to government-gateway-frontend?

We need to keep it consistent with the other redirects to login/create-an-account.
`origin` should not be unknown either I suspect.